app/views/shared/_user_util_links.html.erb

```diff
@@ -0,0 +1,21 @@
+<ul class="navbar-nav">
```

Comment:
Is this an overwrite? If yes, can you please mention what is being overwritten?

Reply:
And I'll add the comment in the same commit.